<launch>
  <arg name="model" default="RS32" />
  <arg name="device_ip" default="192.168.1.200" />
  <arg name="msop_port" default="6699" />
  <arg name="difop_port" default="7788" />
  <arg name="lidar_param_path" default="$(find rslidar_pointcloud)/data/rs_lidar_32/"/>

  <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
    <param name="model" value="$(arg model)"/>
    <param name="device_ip" value="$(arg device_ip)" />
    <param name="msop_port" value="$(arg msop_port)" />
    <param name="difop_port" value="$(arg difop_port)"/>
    <param name="pcap" value="/home/zhwu/Downloads/test_sdkdriver.pcap"/>
    <!-- <param name="pcap" value="/home/zhwu/Downloads/lidar_ros_channel_sel.pcap"/> -->
    
  </node>

  <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
    <param name="model" value="$(arg model)"/>
    <param name="cali_path" value="$(arg lidar_param_path)" />
    <param name="cut_angle" value="0.0" />
    <param name="max_distance" value="200"/>
    <param name="min_distance" value="0.2"/>
    <param name="resolution_type" value="0.5cm"/>
    <param name="intensity_mode" value="1"/>
  </node>

  <node name="rviz" pkg="rviz" type="rviz"  args="-d $(find rslidar_pointcloud)/rviz_cfg/rslidar.rviz" />

</launch>
